### ISSUE TITLE
fails to parse quoted keys in inline tables

### ISSUE THREAD

**Issue by davidszotten** [snyk/snyk-poetry-lockfile-parser#14]
as far as i can tell from reading the toml spec, keys in inline tables are allowed to be quoted, but the toml parser this library is using can't parse that

e.g. adding this to `test/unit/lib/manifest-parser.test.ts`

```
    it('should handle quoted keys in inline tables', () => {
      const fileContents = `[tool.poetry.dependencies]
      pkg_a = {"version" = "^1.0"}`;
      const poetryDependencies = getDependencyNamesFrom(fileContents, false);
      expect(poetryDependencies.length).toBe(1);
      expect(poetryDependencies.includes('pkg_a')).toBe(true);
    });
```

throws exceptions

```
 FAIL  test/unit/lib/manifest-parser.test.ts
  ● when loading manifest files › getDependencyNamesFrom › should handle quoted keys in inline tables

    SyntaxError: Expected "}", [ \t] or [A-Za-z0-9_\-] but "\"" found.
```

**Comment by admons** [comment]
Hey @davidszotten 

letting you know that we've released a new CLI version 1.683.0 that has a fix for this issue
Can you please update your Snyk CLI version to verify this issue and let us know if it worked?

Thanks in advance!

**Comment by davidszotten** [comment]
yep, seems fixed now. thanks!
